docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -129,6 +129,18 @@
       url: /Truncate
     - title: UnderlineNav
       url: /UnderlineNav
+- title: Drafts
```

Comment:
How do you feel about the IA here @siddharthkp? For easier discovery, would it make sense to co-locate it to the original component docs page under an experimental tab/panel/label?

Reply:
Ooh, I like that idea 🤔 

We loose the section heading of "Drafts", but that's probably okay

| Different section        | Collocated           |
| ------------- |-------------|
| ![image](https://user-images.githubusercontent.com/1863771/148949444-f12a5446-64bf-4b46-8ddf-74e67e3c133a.png) | ![image](https://user-images.githubusercontent.com/1863771/148949577-cd93ed3e-7d21-431c-9f7e-3191f297f67c.png) |

Reply:
I thought about it a bit more and decided that we should put all the drafts together. When a draft component is ready to be used in production (design and a11y reviewed, then we should move it out of drafts and deprecate the previous one.)